.github/workflows/icu_ci.yml

```diff
@@ -170,6 +170,25 @@ jobs:
           PYTHON=python3 ./runConfigureICU MacOSX;
           make -j2 check
 
+  # Run ICU4C tests with stubdata.
+  run-with-stubdata:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: ICU4C tests with stubdata
+        run: |
+          cd icu4c/source;
+          ./runConfigureICU Linux;
+          make -j2 check;
+          rm lib/libicudata.so*;
+          cp -P stubdata/libicudata.so* lib;
+          cd test/cintltst;
+          # 'Elapsed Time: ' is printed by makefile upon final success.
```

Comment:
Suggestion: Maybe add an echo so that the two `Elapsed Time: ` output lines at the end are more obviously associated with the stubdata test runs.

Maybe:
```
rm lib/libicudata.so*;
cp -P stubdata/libicudata.so* lib;
cd test/cintltst;
echo 'Running ICU4C cintltst and intltest with stubdata';
```

Otherwise the output at the end is (perhaps) a bit confusing:
```
Elapsed Time: 00:00:14.408
make[2]: Leaving directory '/home/runner/work/icu/icu/icu4c/source/test/cintltst'
---------------
ALL TESTS SUMMARY:
All tests OK:  testdata intltest iotest cintltst
make[1]: Leaving directory '/home/runner/work/icu/icu/icu4c/source/test'
make[1]: Entering directory '/home/runner/work/icu/icu/icu4c/source'
verifying that icu-config --selfcheck can operate
verifying that make -f Makefile.inc selfcheck can operate
PASS: config selfcheck OK
rm -rf test-local.xml
make[1]: Leaving directory '/home/runner/work/icu/icu/icu4c/source'
Elapsed Time: 00:00:03.720
Elapsed Time: 00:00:12.817
```

Reply:
Done.